humanmine.ipynb

```diff
@@ -331,206 +329,412 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "BASE_URL = \"https://idr.openmicroscopy.org/webclient\"\n",
-    "IMAGE_DATA_URL = BASE_URL + \"/render_thumbnail/{id}\"\n",
-    "LINK_URL = BASE_URL + \"/?show=image-{id}\""
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "disease_list = []\n",
+    "disease_list_values = []\n",
+    "gene = \"PDX1\"\n",
```

Comment:
When the gene here is replaced with a gene where no images are found, this cell crashes. Maybe catch and say "no images found for this gene" instead ?